docs/architecture/compaction/compaction-strategies.rst

```diff
@@ -70,7 +70,38 @@ Set the parameters for :ref:`Leveled Compaction <leveled-compaction-strategy-lcs
 Incremental Compaction Strategy (ICS)
 =====================================
 
-ICS is only available in ScyllaDB Enterprise. See the `ScyllaDB Enetrpise documentation <https://enterprise.docs.scylladb.com/stable/architecture/compaction/compaction-strategies.html>`_ for details.
+.. versionadded:: 2019.1.4 Scylla Enterprise
+
+.. note:: ICS is only available for Scylla Enterprise customers
+
+ICS principles of operation are similar to those of STCS, merely replacing the increasingly larger SSTables in each tier, by increasingly longer SSTable runs, modeled after LCS runs, but using larger fragment size of 1 GB, by default.
```

Comment:
If and when we merge the two and deprecate SizeTieredCompactionStrategy (or make it an alias for ICS)
we shouldn't reference STCS anymore, but rather provide full, self-sufficient documentation for ICS.

Reply:
agreed.